src/components/transactionsV2/filters/dateFieldGroup.js

```diff
@@ -250,9 +276,13 @@ class DateFieldGroup extends React.Component {
           </DropdownV2>
           </label>
         </div>
-        <span className={`${styles.feedback} ${this.state.feedback ? styles.show : ''}`}>
-          {this.state.feedback}
-        </span>
+        <Feedback
+          className={styles.feedback}
+          show={!!this.state.feedback}
```

Comment:
here too

Reply:
On this case it would be needed to test both fields error value, this way we only need to check if we have a feedback message for any of the fields.